regression/contracts/quantifiers-nested-05/test.desc

```diff
@@ -5,7 +5,7 @@ main.c
 ^SIGNAL=0$
 ^VERIFICATION SUCCESSFUL$
 --
+^warning: ignoring
```

Comment:
Why is this expected?

Reply:
When CBMC (with SAT backend) does not support a quantifier in a particular context, say forall in a negative context, it prints:

```
warning: ignoring forall
```

So I added these negative regexes to make sure we never hit those cases, otherwise the results would be unsound (since quantifier was completely ignored in the formula).